src/blockdev.rs

```diff
@@ -191,6 +191,14 @@ pub fn reread_partition_table(file: &mut File) -> Result<()> {
             Ok(_) => break,
             Err(err) => {
                 if retries == 0 {
+                    let status = std::process::Command::new("kpartx")
```

Comment:
Could we fallback to this only if we get `Errno::EINVAL`?

Reply:
Done